apps/backend/src/prof-ratings/entities/prof-rating.entity.ts

```diff
@@ -0,0 +1,43 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { IsBoolean, IsDate, IsNotEmpty, IsNumber, IsString, Max, Min } from 'class-validator';
+
+export class ProfRating {
+  @ApiProperty()
```

Comment:
I'm pretty sure @ApiProperty() can be omitted as well.

Putting the @ApiController() decorator to the Controller should be enough. If it isn't, look through our other similar projects (startlap, konzisite) and find the differences

Reply:
Egyelőre úgy sikerült megoldanom, hogy a main.ts-ben betöltöm a generált metadatát, és akkor nem kell az @ApiProperty. Csak az enumokra, azokat még így sem akarta felismerni. Viszont a tanfolyamos ticketing-api-nál ugyanezt a swaggeres package-et használtuk és nem is volt metadata generálva, és így is felismert mindent a swagger. Átolvastam az egész nestes doksit, de sajna nem jöttem rá, hogy mi lehet a baj.

Reply:
hát ez nagyon weird, korábban fixen nem volt szükség ApiProperty()-re meg metadata.ts-re. De akkor ezek szerint most se kéne, hogy legyen, de hirtelen én se látom, mi lehet  a probléma. Arra emlékszem, hogy ezzel én is anno rengeteget szívtam, de az dereng hogy akkor csak nestjs/swagger plugin hiányzott a nest-cli.json-ból, de ez most megvan. Valamikor ránézek majd mélyebben is, de látom te már komoly kutatómunkát csináltál, úgyhogy nem igérek semmi eredményt :D